easybuild/easyconfigs/h/Hyperopt/Hyperopt-0.1-intel-2017b-Python-2.7.14.eb

```diff
@@ -12,7 +12,12 @@ toolchain = {'name': 'intel', 'version': '2017b'}
 
 source_urls = [PYPI_SOURCE]
 sources = [SOURCELOWER_TAR_GZ]
-checksums = ['4f6e903f7640165ea3e4c622050b41ffab0bee7811ede23c7825a5884976d72f']
+patches = ['Hyperopt-0.1_networkx-2.0.patch']
```

Comment:
@smoors Please avoid hardcoding the software version here if it matches: `Hyperopt-%(version)s_networkx-2.0.patch` (and drop the empty line below while you're at it)